### ISSUE TITLE
Windows: Install System Wide

### ISSUE THREAD

**Issue by lhpitn** [ZUGFeRD/quba-viewer#92]
Hi,

we would like to provide Quba Viewer on Our Windows 2019 Terminal-Servers / RemoteDesktop-Session-Hosts, unfortunately executing the exe file or the msi file only installs the Quba app for the local user only. Is there a way to install it system wide for all users?

Regards

TN

**Comment by jstaerk** [comment]
yes, there is a option with the MSI, feel free to have a look at the non existing manual #58  ;-)

**Comment by lhpitn** [comment]
@jstaerk 
it would be grate if you would get asked by the installer (msi) if and how you would like to install it (like on user or system level, which drive, if you want a desktop shortcut or not) like other installers do it

maybe you could leave this open as a feature request?

thanks in advance
